packages/calling-stateful-client/src/CallAgentDeclarative.ts

```diff
@@ -57,6 +65,41 @@ class ProxyCallAgent extends ProxyCallAgentCommon implements ProxyHandler<Declar
     this.unregisterSubscriber();
   };
 
+  protected callDeclaratify(call: CallCommon, context: CallContext): DeclarativeCallCommon {
+    if (isACSCall(call)) {
+      return callDeclaratify(call, context);
+    }
+    throw new Error('Not reachable code for callDeclaratify.');
```

Comment:
Make the error string more actionable: `DaclarativeCallAgent.callDeclaratify must be called with an ACS call`.

same for errors thrown below.